workflow/scripts/reformat-fp-fn-tp-tables.py

```
@@ -16,6 +16,9 @@ def parse_vaf(v):
     return float(v) if isinstance(v, str) else v
 
 def check_samplename_existence(df, sample_name):
+    if snakemake.wildcards.classification == "tp-baseline" or snakemake.wildcards.classification == "fn":
```

Ruff (0.15.6)

[error] Undefined name `snakemake`

(F821)